### ISSUE TITLE
PHP Fatal error:  Class 'Rinvex\Country\CountryLoaderException' not found

### ISSUE THREAD

**Issue by slince** [rinvex/countries#68]
```
.PHP Fatal error:  Class 'Rinvex\Country\CountryLoaderException' not found in E:\www\labs\shipment-tracking\vendor\rinvex\country\src\Loader.php on line 219
```
Version: 2.0.0

I know its has been fixed in v3;  because of my php version, i can only use v2; will you relase a fix version for v2?

**Comment by joshbrw** [comment]
Relates to https://github.com/rinvex/country/issues/43

**Comment by slince** [comment]
@joshbrw  
Hi , I do not quite understand,  how can i solve this problem?

**Comment by joshbrw** [comment]
I believe @Omranic is looking into it.

**Comment by Omranic** [comment]
No idea why this is still open, as it was fixed couple years ago 😅  #43
